src/Client/ClientBase.cpp

```diff
@@ -328,12 +339,16 @@ void ClientBase::initBlockOutputStream(const Block & block, ASTPtr parsed_query)
 
         String current_format = format;
 
+        select_into_file = false;
+
         /// The query can specify output format or output file.
         /// FIXME: try to prettify this cast using `as<>()`
         if (const auto * query_with_output = dynamic_cast<const ASTQueryWithOutput *>(parsed_query.get()))
```

Comment:
Maybe also fix this? :)

Reply:
Most likely it's not possible, because `as` is `typeid_cast`, not `dynamic_cast`.
Let's remove the comment.